sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.institutions.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -112,8 +240,36 @@ describe("AssessmentInstitutionsController(e2e)-getAssessmentNOA", () => {
             firstDisbursementSchedule.disbursementScheduleStatus,
           disbursement1TuitionRemittance:
             firstDisbursementSchedule.tuitionRemittanceRequestedAmount,
-          ...awards,
+          ...firstDisbursementScheduleAwards,
+          disbursement2COEStatus: secondDisbursementSchedule.coeStatus,
+          disbursement2Date: getDateOnlyFormat(
+            secondDisbursementSchedule.disbursementDate,
+          ),
+          disbursement2Id: secondDisbursementSchedule.id,
+          disbursement2MSFAACancelledDate:
+            secondDisbursementSchedule.msfaaNumber?.cancelledDate,
+          disbursement2MSFAADateSigned:
+            secondDisbursementSchedule.msfaaNumber?.dateSigned,
+          disbursement2MSFAAId: secondDisbursementSchedule.msfaaNumber?.id,
+          disbursement2MSFAANumber: MASKED_MSFAA_NUMBER,
+          disbursement2Status:
+            secondDisbursementSchedule.disbursementScheduleStatus,
+          disbursement2TuitionRemittance:
+            secondDisbursementSchedule.tuitionRemittanceRequestedAmount,
+          ...secondDisbursementScheduleAwards,
         },
+        eligibleAmount: assessment.disbursementSchedules.reduce(
+          (totalValueAmount, schedule) => {
+            return (
+              totalValueAmount +
+              (schedule.disbursementValues || []).reduce(
+                (sum, disbursementValue) => sum + disbursementValue.valueAmount,
+                0,
```

Comment:
https://github.com/bcgov/SIMS/pull/2365/files#r1346581248

Reply:
Removed it